wsrep_loader.c

```diff
@@ -165,7 +166,13 @@ int wsrep_load(const char *spec, wsrep_t **hptr, wsrep_log_cb_t log_cb)
         return ret;
     }
 
-    if (!(dlh = dlopen(spec, RTLD_NOW | RTLD_LOCAL))) {
+#ifdef __SANITIZE_ADDRESS__
```

Comment:
To improve readability, consider placing this code close to the declaration and initialization of `open_flags` variable. Or move declaration of `open_flags` variable down here.